e2etest/utils.go

```diff
@@ -25,3 +30,26 @@ func runCommand(bgp bool, name string, args ...string) (string, error) {
 	out, err := exec.Command(name, args...).CombinedOutput()
 	return string(out), err
 }
+
+func wgetRetry(bgp bool, address string) error {
+	retrycnt := 0
+	code := NetworkFailure
+	var err error
+
+	// Retry loop to handle wget NetworkFailure errors
+	for {
+		_, err = runCommand(bgp, "wget", "-O-", "-q", address, "-T", "60")
+		if err != nil && retrycnt < retryLimit && code == NetworkFailure {
```

Comment:
I'm a little confused by `code == NetworkFailure` here.  This will *always* be true the first time through the loop.  We extract the exit code *after* the check, but then don't check it.  On the 2nd (or later) runs through, we're checking the exit code of the previous execution of wget.  I suspect this isn't what you intended?

Reply:
yeah u are right logic wasn't exactly what I wanted I rearranged hopefully will be ok now